Source/Plugins/Core/com.equella.core/src/com/tle/web/api/search/EquellaSearchResource.java

```diff
@@ -127,11 +128,11 @@
           @QueryParam("dynacollection")
           String dynaCollectionCompound,
       @ApiParam(
-              value = "Filter by item status",
-              required = false,
-              allowableValues =
-                  "DRAFT,LIVE,REJECTED,MODERATING,ARCHIVED,SUSPENDED,DELETED,REVIEW,PERSONAL",
-              allowMultiple = true)
+              value =
+                  "Filter by item status"
+                      + "\nAvailable values: "
+                      + "DRAFT,LIVE,REJECTED,MODERATING,ARCHIVED,SUSPENDED,DELETED,REVIEW,PERSONAL",
```

Comment:
Seeing this is now doco, let's make sure we include spaces:

```suggestion
                      + "DRAFT, LIVE, REJECTED, MODERATING, ARCHIVED, SUSPENDED, DELETED, REVIEW, PERSONAL",
```

But also, I note above (for `info`) you've used the enum directly. Is that an option here? But the other thing is, in one you're saying the values should be lower case and the other upper case - be good to be consistent.

Reply:
No, but I can add one.